basis/io/directories/directories.factor

```diff
@@ -68,20 +68,12 @@ HOOK: delete-directory io-backend ( path -- )
 : to-directory ( from to -- from to' )
     over file-name append-path ;
 
-! Moving and renaming files
-HOOK: move-file io-backend ( from to -- )
-
-: move-file-into ( from to -- )
-    to-directory move-file ;
-
-: move-files-into ( files to -- )
-    '[ _ move-file-into ] each ;
-
 ! Copying files
 HOOK: copy-file io-backend ( from to -- )
 
 M: object copy-file
-    make-parent-directories binary <file-writer> [
+    [ normalize-path ] bi@ make-parent-directories
```

Comment:
It was in the Windows copy-file implementation, for which this code is now a replacement.
I don't know if it *should* be there, but it was.